src/Spout/Writer/XLSX/Helper/StyleHelper.php

```diff
@@ -110,13 +110,21 @@ protected function getBordersSectionContent()
             $border = $style->getBorder();
             if ($border) {
                 $content .= '<border>';
-                foreach ($border->getParts() as $part) {
-                    /** @var $part \Box\Spout\Writer\Style\BorderPart */
-                    $content .= BorderHelper::serializeBorderPart($part);
+
+                $sortOrder = ['left', 'right', 'top', 'bottom'];
```

Comment:
Can you add a comment saying why they need to be in this order (or just add the link to the issue)